src/Compilers/CSharp/Portable/DocumentationComments/DocumentationCommentIDVisitor.PartVisitor.cs

```diff
@@ -270,6 +270,20 @@ public override object VisitDynamicType(DynamicTypeSymbol symbol, StringBuilder
                 return null;
             }
 
+            private object VisitTupleType(TupleTypeSymbol symbol, StringBuilder builder)
+            {
+                // get the underlying tuple type. 
+                var tupleType = symbol.UnderlyingNamedType;
```

Comment:
>tupleType [](start = 20, length = 9)

nit: in terms of naming convention, we call `(....)` "tuples", and their underlying type "ValueTuple". Consider naming the local "valueTupleType" or "underlyingType", as the underlying type is not itself a tuple.